pandas/tests/test_accent.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+from pandas import read_csv, DataFrame
+from pandas.util.testing import ensure_clean
+
+class FileSystemEncodingTest(unittest.TestCase):
+    """Test compatibility with file system encoding"""
```

Comment:
needs to go with tests in pandas/io/tests/parser

try to find a good place there